src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -154,8 +270,22 @@ def _get_molecular_data(self):
 
         pyscf_molecular_data = PyscfMolecularData.__new__(PyscfMolecularData)
         pyscf_molecular_data.__dict__.update(molecule.__dict__)
+
         return molecular_data
 
+    def _get_frozen_core_orbitals(self, molecular_data) -> Tuple[mp.mp2.MP2, int]:
+        """
+        Get auto-generated chemical core orbitals.
+
+        Args:
+            molecular_data: PyscfMolecularData object.
+        Returns
+            The mp2 object and the number of chemical core orbitals.
+
+        """
+        mp2 = mp.MP2(molecular_data._pyscf_data["scf"]).set_frozen()
+        return mp2, mp2.frozen
```

Comment:
Since `mp2.frozen` is derived from `mp2`, we don't have to return it separately.
It's enough to return `mp2` and then extract `mp2.frozen` wherever we need it – one return object is better than two.

Reply:
Good point! I removed `mp2.frozen`.